README.md

```diff
@@ -1,128 +1,130 @@
+- [Github Issues](https://github.com/pythonindia/inpycon2020-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Fplatform)
 - Members
-  * Vipul Siddharth (C)
-  * Chandan Kumar
-  * Ananyo Maiti
+  * Chandan Kumar (C)
   * Sayan Chowdhury
-  
-### Review Panel
 
-- [Github Issues](https://github.com/pythonindia/inpycon2020-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Fmentorship)
+### Review
+
+- [Github Issues](https://github.com/pythonindia/inpycon2020-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Freview)
+- Chair: Ritesh Agarwal
 - Members
-  * Ritesh Agarwal (C)
   * Neha Kandpal
   * Sayan Chowdhury
-  
-### Content
 
-- [Github Issues](https://github.com/pythonindia/inpycon-blog/issues/)
+### Social Media
+
+- [Github Issues](https://github.com/pythonindia/inpycon2020-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Fsocial-media)
+- Chair: Pradhvan Bisht
+- Co-chairs: Rayan Das, Abhishek Mishra
 - Members
-  * Vipul Gupta (C)
-  * Nikhil Maan
+  * TBD
 
-### Platform
+### Sponsorship
 
-- [Github Issues](https://github.com/pythonindia/inpycon2020-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Fmentorship)
-- Members
-  * Chandan Kumar (C)
-  * Sayan Chowdhury
+- [Github Issues](https://github.com/pythonindia/inpycon2020-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Fsponsorship)
+- Chair: Sayan Chowdhury (@sayanchowdhury)
+- Co-chair: TBD
 
 ### Technology
 
 - [Github Issues](https://github.com/pythonindia/inpycon2020-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Ftechnology)
+- Chair: Nabarun Pal
 - Members
-  * Nabarun Pal (C)
-  * Saptak Sengupta (CC)
   * Gaurav Kumar
   * Sayan Chowdhury
+  * Abhishek Mishra
 
-### Diversity
-
-- Chair: TBD
-- Co-Chair: TBD
-- [Github Issues](https://github.com/pythonindia/inpycon2020-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Fdiversity)
-
+## WGs being formed
 
-## To be decided later
 ### Billing
 
 - Chair: TBD
 - Co-Chair: TBD
 - [Github Issues](https://github.com/pythonindia/inpycon2020-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Fbilling)
 
-### Sponsorship
+### Diversity
 
-- Chair: Sayan Chowdhury (@sayanchowdhury)
+- Chair: TBD
 - Co-Chair: TBD
-- [Github Issues](https://github.com/pythonindia/inpycon2020-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Fsponsorship)
-
+- [Github Issues](https://github.com/pythonindia/inpycon2020-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Fdiversity)
 
 ### Tickets
 
 - Chair: TBD
 - Co-Chair: TBD
 - [Github Issues](https://github.com/pythonindia/inpycon2020-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Ftickets)
 
+## Committees
+
+### Code of Conduct
+
+- [Github Issues](https://github.com/pythonindia/inpycon2020-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Fcode-of-conduct)
+- Head: Sayan Chowdhury
+- Members
+  * Saptak Sengupta
+  * Neha Kandpal
+
 ## Timeline
 ### TLDR;
 * Sponsership going live on 31st July, 2020.
```

Comment:
Sponsership -> Sponsorship

Reply:
Ah! This was there earlier. Let me fix it.